src/gui/mzroll/background_peaks_update.cpp

```diff
@@ -485,54 +414,6 @@ void BackgroundPeakUpdate::align() {
                         aligner.doAlignment(groups);
                         mainwindow->sampleRtWidget->setDegreeMap(aligner.sampleDegree);
                         mainwindow->sampleRtWidget->setCoefficientMap(aligner.sampleCoefficient);
-                } else if (alignAlgo == 2) {
-                        mainwindow->getAnalytics()->hitEvent("Alignment", "LoessFit");
-                        aligner.preProcessing(groups, mavenParameters->alignWrtExpectedRt);
-                        // initialize processedDataFromPython with null 
-                        processedDataFromPython="";
-                         /**runPythonProg()
-                         * sends the json of groups and samples rt to the python exe. for more look in sendDataToPython()
-                         * python exe is going to correct the rts and send it back to us in json format
-                         */
-                        runPythonProg(&aligner);
-                        // wait for processing of data by python program
-                        pythonProg->waitForFinished(-1);                        
-
-                        // convert the data to json
-                        QJsonDocument jDoc;
-                        QJsonObject parentObj;
-
-                        // if jDoc is null that means the json returned from python is malformed
-                        // in such a case our rts wont update with new values
-                        jDoc = QJsonDocument::fromJson(processedDataFromPython);
-
-                        QString errorMessage=QString::number(processedDataFromPython.size());
-
-                        if(!jDoc.isNull()){
-                                parentObj = jDoc.object();
-                        }
-                        else{
-                                if(processedDataFromPython.size()==0){
-                                        errorMessage=errorMessage + " good groups found." +"<br>"+"Relax parameters for better result";
-                                }
-                                else{
-                                        errorMessage=errorMessage+"<br>"+"Incomplete data, re-run alignment";
-                                }
-                                qDebug()<<errorMessage;
-                                Q_EMIT alignmentError(errorMessage);
-                                return;
-                        }
-
-                        if(!parentObj.isEmpty()){
-                                aligner.updateRts(parentObj);
```

Comment:
`Aligner::updateRts` is not being used anywhere and can be removed

Reply:
Deleted these as well.